--- orgsources/manage_user_edit_page.php	Mon Nov 12 00:10:40 2012
+++ manage_user_edit_page.php	Sun Jan 06 14:37:30 2013
@@ -289,6 +289,9 @@
 <?php
 	} # End of PROJECT ACCESS conditional section
 
+	// event siganl for adding userdefinitions by plugins
+	event_signal( 'EVENT_MANAGE_USER_FORM'); 
+	
 	include ( 'account_prefs_inc.php' );
 	edit_account_prefs( $t_user['id'], false, false, 'manage_user_edit_page.php?user_id=' . $t_user_id );
 
